Tests/SonarScanner.MSBuild.PreProcessor.Test/PreProcessorTests.cs

```diff
@@ -125,6 +125,8 @@ public void PreProc_EndToEnd_SuccessCase()
             logger.DebugMessages.Should().Contain("Base branch parameter was not provided. Incremental PR analysis is disabled.");
```

Comment:
clean as you code: could you please update also this line to use `logger.AssertDebugLogged`?